assembl/static2/js/app/components/debate/brightMirror/fictionHeader.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React, { Fragment } from 'react';
+
+import CircleAvatar from './circleAvatar';
+import type { CircleAvatarType } from './circleAvatar';
+
+export type FictionHeaderType = {
+  /** Author fullname */
+  authorFullname: string,
+  /** Article published date, format is yyyy-mm-dd */
+  publishedDate: Date,
+  /** Circle avatar props */
+  circleAvatar: CircleAvatarType
+};
+
+const noAuthorMessage: string = 'no author specified';
+
+const fictionHeader = ({ authorFullname, publishedDate, circleAvatar }: FictionHeaderType) => {
+  const formattedPublishedDate = publishedDate.toISOString().slice(0, 10);
+
+  return (
+    <Fragment>
+      <header className="header">
+        <CircleAvatar {...circleAvatar} />
+        <div className="meta">
+          <p className="author">{authorFullname || noAuthorMessage}</p>
+          <p className="date-time">
+            Le{' '}
+            <time dateTime={formattedPublishedDate} pubdate="true">
+              {formattedPublishedDate
+                .split('-')
+                .reverse()
+                .join('/')}
```

Comment:
You should internationalize the same way we do in other places.
{I18n.l(post.creationDate, { dateFormat: 'date.format2' })}

To add "Le", it's complicated... do we really need it?

Reply:
@vincentfretin should ask Roxanna, it's on the mockup and should/will be i18n

Reply:
Fixed + remove the "Le"